operations/service_web.go

```diff
@@ -134,12 +134,19 @@ func startWebService() cli.Command {
 	}
 }
 
-func gracefulShutdownForSIGTERM(ctx context.Context, servers []*http.Server, wait chan struct{}, catcher grip.Catcher) {
+func gracefulShutdownForSIGTERM(ctx context.Context, servers []*http.Server, wait chan struct{}, catcher grip.Catcher, env evergreen.Environment) {
 	defer recovery.LogStackTraceAndContinue("graceful shutdown")
 	sigChan := make(chan os.Signal, len(servers))
 	signal.Notify(sigChan, syscall.SIGTERM)
 
 	<-sigChan
+	// we got the signal, so modify the status endpoint and wait
+	err := env.SetShutdown()
+	if err != nil {
+		grip.Error(err)
```

Comment:
I suggested elsewhere to remove the error value altogether, but FYI you wouldn't need the if statement even if `env.SetShutdown()` could return an error, since `grip.Error(nil)` is a noop.